Packs/FiltersAndTransformers/Scripts/RemoveMatches/RemoveMatches.yml

```diff
@@ -0,0 +1,48 @@
+args:
+- default: true
+  description: The value on which the transformer is applied.
+  isArray: true
+  name: value
+- auto: PREDEFINED
+  defaultValue: "Yes"
+  description: Whether to ignore the case of the item for which you are searching. Default is "Yes".
+  name: ignore_case
+  predefined:
+  - "yes"
+  - "no"
+- auto: PREDEFINED
+  defaultValue: "No"
+  description: Whether to match the exact item in the list, or look for any string that contains it. Default is "No".
+  name: match_exact
+  predefined:
+  - "yes"
+  - "no"
+- defaultValue: \n
+  description: A string used to delimit fields. For example, a new line "\n" should match the list separator configuration.
+  name: delimiter
+- description: A list of patterns to remove from the value. This can be a single string or a list of patterns, separated by the pattern defined in the delimiter Argument. Unless match_exact is yes, Regex pattern are supported.
```

Comment:
```suggestion
- description: A list of patterns to remove from the value. This can be a single string or a list of patterns, separated by the pattern defined in the delimiter argument. Unless match_exact is yes, regex pattern is supported.
```